drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/events/ScenarioNotificationEvent.java

```diff
@@ -27,11 +27,19 @@
     public static final Type<ScenarioNotificationEventHandler> TYPE = new Type<>();
 
     private final String message;
+    private final boolean autoHide;
     private final org.uberfire.workbench.events.NotificationEvent.NotificationType notificationType;
 
     public ScenarioNotificationEvent(String message, NotificationEvent.NotificationType type) {
         this.message = message;
         this.notificationType = type;
+        this.autoHide = true;
+    }
+
+    public ScenarioNotificationEvent(String message, boolean autoHide, NotificationEvent.NotificationType type) {
```

Comment:
I would maybe keep the additional argument (the one distinguishing the overloaded method) as the last one.

Reply:
@jstastny-cz Done.